compose/ui/ui/src/skikoMain/kotlin/androidx/compose/ui/scene/BaseComposeScene.skiko.kt

```diff
@@ -145,27 +146,31 @@ internal abstract class BaseComposeScene(
         recomposer.performScheduledTasks()
     }
 
-    override fun render(canvas: Canvas, nanoTime: Long) = postponeInvalidation {
-        // Note that on Android the order is slightly different:
-        // - Recomposition
-        // - Layout
-        // - Draw
-        // - Composition effects
-        // - Synthetic events
-        // We do this differently in order to be able to observe changes made by synthetic events
-        // in the drawing phase, thus reducing the time before they are visible on screen.
-        //
-        // It is important, however, to run the composition effects before the synthetic events are
-        // dispatched, in order to allow registering for these events before they are sent.
-        // Otherwise, events like a synthetic mouse-enter sent due to a new element appearing under
-        // the pointer would be missed by e.g. InteractionSource.collectHoverAsState
-        recomposer.performScheduledTasks()
-        frameClock.sendFrame(nanoTime)           // Recomposition
-        doLayout()                               // Layout
-        recomposer.performScheduledEffects()     // Composition effects (e.g. LaunchedEffect)
-        inputHandler.updatePointerPosition()     // Synthetic move event
-        snapshotInvalidationTracker.onDraw()
-        draw(canvas)                             // Draw
+    override fun render(canvas: Canvas, nanoTime: Long) {
+        return trace("BaseComposeScene:render") {
+            postponeInvalidation {
```

Comment:
Let's add `trace` inside `postponeInvalidation` and pass tag as parameter

Reply:
Added trace, but didn't get what you mean under `pass tag as a parameter`, can you elaborate?